js/deer-config.js

```diff
@@ -314,7 +319,7 @@ export default {
                         }
                         for(const prop in query){
                             if(li.hasAttribute(`data-${prop}`)){
-                                const action = li.getAttribute(`data-${prop}`).includes(query[prop]) ? "remove" : "add"
+                                const action = li.getAttribute(`data-${prop}`).toLowerCase().includes(query[prop]) ? "remove" : "add";
```

Comment:
You could make the code a lot simpler.  Just make `query[prop]` lowercase here.  That way, you don't need to alter the `query` data itself.